### PR TITLE
[RHPAM-1829] - User password is been exposed in the Readiness probe a…

### PR DESCRIPTION
**Pull request by spolti**
…nd Liveness probe

### CODE REVIEW

**Comment by kiereleaseuser2** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

File: kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/WEB-INF/web.xml [hunk 1]
```diff
@@ -22,6 +22,15 @@
...
     <!-- No authentication. Swagger resources should be accessible without authentication and files (form rendering). -->
   </security-constraint>
+  <security-constraint>
+    <web-resource-collection>
+      <web-resource-name>KIE Server Health and Ready Probes</web-resource-name>
+      <url-pattern>/services/rest/server/readycheck</url-pattern>
+      <url-pattern>/services/rest/server/healthcheck</url-pattern>
+      <http-method>GET</http-method>
+    </web-resource-collection>
+    <!-- No authentication KIE Server health and  -->
```
Comment (errantepiphany) [inline kie-server-parent/kie-server-wars/kie-server/src/main/ee7-resources/WEB-INF/web.xml]: ...health and "readiness"

**Comment by mswiderski** [comment]
@spolti could you please rebase as there is conflict now. And please add the same changes to webc/WEB-INF/web.xml so we have it consistent.

**Comment by spolti** [comment]
@mswiderski done, thanks.

**Comment by mswiderski** [comment]
Ok to test

**Comment by mswiderski** [comment]
Jenkins retest this

**Comment by mswiderski** [comment]
Jenkins retest this

**Pull request merged**